### PR TITLE
Adding Color Presets

### PR DESCRIPTION
**Pull request by adrian1001**
Added a small color palette, accessible also via keyboard shortcuts 0..9.
Can be deactivated by setting an empty list of preset colors.

Use case:
When doing a joint whiteboard session in a team, it improves the visual result if everybody sticks to a small number of predefined colors rather than picking from the whole palette.

### CODE REVIEW

File: client-data/js/board.js [hunk 1]
```diff
@@ -26,6 +26,15 @@
...
 
+Tools.i18n = (function i18n() {
+	var translations = JSON.parse(document.getElementById("translations").text);
+	return {
+		"t": function translate(s) {
+			return translations[s] || s;
+		}
+	};
+})();
+
```
Comment (adrian1001) [inline client-data/js/board.js:37]: Moved this up here, because I got a not-defined warning in Firefox for a line where Tools.i18n was referenced

**Comment by lovasoa** [comment]
Hey @adrian1001  ! I wanted to rebase this PR against the latest master, but I accidentally pushed the wrong commit to your repo, which closed this PR, and removed my permission to push to your fork. Can you please reopen it ? I'll finish the rebase and we can then discuss your changes.

**Comment by lovasoa** [comment]
About the PR itself: 

It looks very good to me, I'm ready to merge it. But I'd like to remove the use of non-essential modern javascript features. The code currently does not go through any compiler, and is compatible even with ancient browsers. I'd like to update the code a little to replace the `Map` with a simple array of arrays.

Also: the CSS changes seem to make the tools' tooltip opening animation look weird, can we fix that ?

**Comment by lovasoa** [comment]
@adrian1001 , if you are okay with my changes in #25, I'll merge them.

**Pull request closed without merging**